pkgs/by-name/ve/version-cli/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+buildGoModule rec {
```

Comment:
Lets remove this `rec` in favor of using `buildNpmPackage (finalAttrs: { ... })`. This pattern allow the values to be properly updated when applying packages override